packages/travix-ui-kit/components/tabs/tab.css

```diff
@@ -1,76 +1,67 @@
 .ui-tab {
+  background-color: var(--tx-generic-color-blank);
+  box-sizing: content-box;
+  color: var(--tx-generic-color-primary-dark);
   cursor: pointer;
-  flex: 1 1 auto;
-  padding: 4px 0 0;
-  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 0px; /* flex-basis: Opx is mandatory for IE */
```

Comment:
Is the unit also required?

Reply:
yes. `flex: 1 0 0;` does not work in ie. Please let me know if you know how to make my comment more clear